castle.json

```diff
@@ -0,0 +1 @@
333110957, -1061.8624340778413], [-297.0946799780435, -1060.7533198959975], [-297.0430932719112, -1059.611964022821], [-296.9657132127127, -1058.4383664583117], [-296.8625398004482, -1057.2325272024698], [-296.73357303511756, -1055.994446255295], [-296.5610799864878, -1054.754753223554], [-296.3450606545589, -1053.513448107246], [-296.08551503933086, -1052.2705309063717], [-295.7824431408037, -1051.0260016209304], [-295.4358449589775, -1049.7798602509229], [-295.04572049385223, -1048.5321067963482], [-294.6120697454278, -1047.2827412572074], [-294.13489271370423, -1046.0317636334998], [-293.6141893986816, -1044.7791739252257], [-293.0499598003599, -1043.5249721323848], [-292.44220391873904, -1042.2691582549771], [-291.7909217538191, -1041.0117322930028], [-291.0961133056, -1039.752694246462], [-290.3577785740818, -1038.4920441153545], [-289.57591755926455, -1037.2297818996803], [-288.7505302611481, -1035.96590759944], [-287.9219187938985, -1034.7520079207648], [-287.09008315751555, -1033.5880828636555], [-286.2550233519994, -1032.4741324281115], [-285.4167393773499, -1031.4101566141333], [-284.57523123356725, -1030.3961554217208], [-283.73049892065126, -1029.4321288508738], [-282.882542438602, -1028.5180769015926], [-282.0313617874195, -1027.6539995738774], [-281.1769569671037, -1026.8398968677275], [-280.31932797765467, -1026.075768783143], [-279.4584748190723, -1025.3616153201244], [-278.59439749135674, -1024.6974364786713], [-277.72709599450786, -1024.083232258784], [-276.8565703285258, -1023.5190026604621], [-275.9828204934104, -1023.004747683706], [-275.10584648916165, -1022.5404673285157], [-274.2256483157797, -1022.1019803263913], [-273.3422259732646, -1021.6892866773331], [-272.45557946161614, -1021.3023863813411], [-271.56570878083437, -1020.9412794384151], [-270.6726139309194, -1020.6059658485553], [-269.77629491187116, -1020.2964456117616], [-268.8767517236896, -1020.0127187280341], [-267.97398436637485, -1019.7547851973727], [-267.0679928399268, -1019.5226450197774], [-266.15877714434544, -1019.3162981952485], [-265.24633727963084, -1019.1357447237855], [-264.330673245783, -1018.9809846053887], [-263.41178504280185, -1018.852017840058], [-262.48967267068747, -1018.7488444277934], [-261.5643361294398, -1018.671464368595], [-260.63577541905886, -1018.6198776624626], [-259.7039905395447, -1018.5844118019967], [-258.7689814908972, -1018.5650667871971], [-257.83074827311646, -1018.5618426180639], [-256.8892908862025, -1018.574739294597], [-255.94460933015523, -1018.6037568167964], [-254.99670360497464, -1018.648895184662], [-254.04557371066085, -1018.7101543981942], [-253.09121964721385, -1018.7875344573926], [-252.07238220110145, -1018.8939320387904], [-250.98906137232365, -1019.0293471423877], [-249.84125716088056, -1019.1937797681842], [-248.62896956677213, -1019.3872299161803], [-247.1684209494021, -1019.648387615975], [-245.4596113087705, -1019.977252867568], [-242.95120772308866, -1020.4898957597576]]], "color": "pink", "y": 192.0}]}
```

Comment:
@savage13 just noticed that the names do not match the JSON in the Radar repo. Could you provide an updated version of this? (and the generator script if you still have it – ideally it should be part of the repo)

Reply:
I did the digitization (polygons) and location naming by hand. :|  I was also wandering around in the game trying to find objects. This is the reason for the name mismatches and some missing polygons.

I should be able to construct a simple script that filters the the objects through the polygon file and assigns them locations based which region they are in.  

This should solve the mismatch between the polygon names and the radar locations.  

My apologies for the churn on this and thanks for you patience.